flink-optimizer/src/test/java/org/apache/flink/optimizer/testfunctions/IdentityFilter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.optimizer.testfunctions;
+
+import org.apache.flink.api.common.functions.FilterFunction;
+
+/**
+ * FilterFunction for optimizer tests.
+ */
+public class IdentityFilter<T> implements FilterFunction<T> {
```

Comment:
How about just using lambdas (filter always works) instead of adding this class?

Reply:
Good point. 
Didn't think about that as the other tests follow that pattern.
Will remove the class.